game-of-life-multiplatform/web/webpack.common.js

```diff
@@ -7,7 +7,7 @@ module.exports = {
         filename: "bundle.js"
     },
     resolve: {
-        modules: [path.resolve(__dirname, "node_modules"), path.resolve(__dirname, "build/classes/main/min/")]
+        modules: [path.resolve(__dirname, "node_modules"), path.resolve(__dirname, "build/kotlin-js-min/main/")]
```

Comment:
This is unrelated to the IOS client, right? Is it needed due to the plugin version update?

Reply:
Plugin version + Gradle version. The old JS setup we copied from kotlinconf app does not work with Gradle 4.+ and new native multi-platform does not work with older versions than Gradle 4.0.

Reply:
Here is the commit https://github.com/novoda/spikes/pull/366/commits/467c4521d374441bb20fe13476812b03181298c2